tests/test_routers.py

```diff
@@ -466,6 +486,26 @@ def test_regex_url_path_detail(self):
         assert json.loads(response.content.decode()) == {'pk': pk, 'kwarg': kwarg}
 
 
+@unittest.skipUnless(path, 'django.urls.path is not defined')
+class TestUrlPath(URLPatternsTestCase, TestCase):
+    urlpatterns = [
+        path('path/', include(url_path_router.urls))
+    ] if path else []
+
+    def test_url_path_list(self):
```

Comment:
I might rename these to `test_list_extra_action` and `test_detail_extra_action` to differentiate them from tests on the regular CRUD list/detail routes.